compiler/rustc_lint/src/late.rs

```diff
@@ -429,7 +429,7 @@ fn late_lint_crate_inner<'tcx, T: LateLintPass<'tcx>>(
 /// Performs lint checking on a crate.
 pub fn check_crate<'tcx, T: LateLintPass<'tcx> + 'tcx>(
     tcx: TyCtxt<'tcx>,
-    builtin_lints: impl FnOnce() -> T + Send,
+    builtin_lints: impl FnOnce() -> T + Send + DynSend,
```

Comment:
Are both `Send` and `DynSend` necessary?

Reply:
`Send` it the origin bound of `builtin_lint`, so I keeped it here

The compiler cannot deduce that an instance of `Send` is also `DynSend`, so we added the bound here